environment-api/src/main/java/com/sequenceiq/environment/api/environment/model/response/DatalakePrerequisiteV1Response.java

```diff
@@ -0,0 +1,55 @@
+package com.sequenceiq.environment.api.environment.model.response;
+
+import java.io.Serializable;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.sequenceiq.environment.api.LdapV4Response;
+import com.sequenceiq.environment.api.environment.doc.EnvironmentModelDescription;
+
+import io.swagger.annotations.ApiModelProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(Include.NON_NULL)
+public class DatalakePrerequisiteV1Response implements Serializable {
+
+    @NotNull
+    @ApiModelProperty(EnvironmentModelDescription.LDAP_RESPONSE)
+    private LdapV4Response ldap;
+
+    @ApiModelProperty(EnvironmentModelDescription.RDS_RESPONSE)
+    private Set<String> databases = new HashSet<>();
+
+    @NotNull
+    @ApiModelProperty(EnvironmentModelDescription.KERBEROS_RESPONSE)
+    private String kerberos;
+
+    public LdapV4Response getLdap() {
```

Comment:
I guess this is the same case, what you told me about secrets. Right?

Reply:
yes, it is the same. Tamas finished the LDAP, so it will be renamed